client/Packages/com.beamable/Editor/BuildPreProcessor.cs

```diff
@@ -141,12 +163,12 @@ Please make sure you have a config-defaults.txt file in Assets/Beamable/Resource
 			if (!cidsMatch || !pidsMatch)
 			{
 				warningMessage = $@"BEAMABLE WARNING: CID/PID Mismatch Detected!
-The editor environment is using a cid=[{editorCid}] and pid=[{editorPid}]. These values are assigned in Toolbox.
-However, the built target will use a cid=[{runtimeCid}] and pid=[{runtimePid}]. These values are assigned in the config-defaults.txt file.
+The editor environment is using a <b>cid=[{editorCid}]</b> and <b>pid=[{editorPid}]</b>. These values are assigned in Toolbox.

```

Comment:
`These values are assigned in Toolbox.` Can we take the chance and replace it since Toolbox was removed in 2.0?

Reply:
Or the toolbox now means the `Beamable button` on top?

Reply:
I would change it to `These values are accessible from Beamable Menu in the Select Realm section.`.